packages/gatsby-plugin-manifest/README.md

```diff
@@ -52,4 +57,62 @@ plugins: [
 ];
 ```
 
+### Hybrid configuration
```

Comment:
Does including a "hybrid" configuration make sense? Wouldn't you just do one or the other? Or is the idea that you could want different icons at larger sizes?

Reply:
Yeah. It was going to be more code to eliminate this option then it was to leave it in. I figured it was useful if people wanted to not create all the icons or wanted to create different or larger icons. It's also useful if they just didn't like my naming conventions and wanted to modify path or name. I also believe it'd allow them to use webp or jpeg instead of png if they so desired (I believe the default output image is set to match the input. So if they gave it a jpeg it'd spit out a jpeg). 

Basically it gives more options and without adding any code complications and little to no config complications.